adaptive_scheduler/kernel/reservation.py

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env python
 '''
-Reservation_v3 and CompoundReservation_v2 classes for scheduling.
+Reservation and CompoundReservation classes for scheduling.
```

Comment:
Most of the changes in here are just refactoring to remove old code and change names and add variables.